src/main/com/example3/radek/ArrayRNGApplication.java

```diff
@@ -0,0 +1,33 @@
+package main.com.example3.radek;
+
+import java.security.SecureRandom;
+import java.util.Arrays;
+
+public class ArrayRNGApplication {
+
+    public static void main(String[] args) {
+
+        int[] array = new int[49];
+        int[] array2 = new int[6];
+        array = getFilledArray(false, array);
+        Arrays.stream(array).boxed().forEach(result -> System.out.print(result + " "));
```

Comment:
Tak ty si nedas pokoj s tou lambdou jo? :D 

Radky 12 a 15 naprosto stejne --> metoda s jednim vstupnim parametrem.

Reply:
Dal bych si s ní pomoct, ale u tohoto případu vím, že pro každý result -> udělej tuto akci :D nepodařilo se mi napsat forEach aby udělal print + mezeru, ale jen print. S lambdou jsem to dal a i pochopil ten syntax

Reply:
printArray(array) metoda je hotová